features/support/feature_tests/action/login.rb

```diff
@@ -0,0 +1,33 @@
+module FeatureTests
+  module Action
+    module Login
+      extend Capybara::DSL
+      extend RSpec::Matchers
```

Comment:
I had this, but the tests didn't find the new `FeatureTests::Helper` module when loading all the files.

Dumb reason not to do this, but we were not able to solve the loading problem with @rap1ds and decided to leave the test helpers as they were.